### ISSUE TITLE
Comboboxes are not accessible using keyboard in the profile editor

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#3439]
For example, these two cannot be edited using just keyboard (by TAB-ing through the UI)

![image](https://github.com/user-attachments/assets/f7a07bd9-c4f2-45db-a9e6-c9a3ada26384)

![image](https://github.com/user-attachments/assets/762ddb01-8e30-4b53-a50c-731e7e53c348)

**Comment by alexanderbock** [comment]
Might have been updated in https://github.com/OpenSpace/OpenSpace/pull/3417

**Comment by WeirdRubberDuck** [comment]
Yes, this has been fixed in that PR.
